api/app/serializers/spree/api/v2/platform/promotion_action_line_item_serializer.rb

```diff
@@ -0,0 +1,14 @@
+module Spree
+  module Api
+    module V2
+      module Platform
+        class PromotionActionLineItemSerializer < BaseSerializer
+          include ResourceSerializerConcern
+          attribute :variant_id
```

Comment:
@MatthewKennedy I think it would be better to add an association to Variant here

Reply:
I'll sort this.